ai2thor/tests/test_unity.py

```diff
@@ -1722,6 +1712,24 @@ def test_unsupported_manipulathor(controller):
     assert not event, "PickupObject(objectId) should have failed with agentMode=arm"
 
 
+@pytest.mark.parametrize("controller", fifo_wsgi)
+def test_set_random_seed(controller):
+    f1_1 = controller.reset().frame
+    f1_2 = controller.step(action="SetRandomSeed", seed=42).frame
+    f1_3 = controller.step(action="RandomizeMaterials").frame
+
+    f2_1 = controller.reset().frame
+    f2_2 = controller.step(action="SetRandomSeed", seed=42).frame
+    f2_3 = controller.step(action="RandomizeMaterials").frame
+
+    assert_images_near(f1_1, f2_1)
+    assert_images_near(f1_1, f1_2)
+    assert_images_near(f2_1, f2_2)
+    assert_images_near(f1_3, f2_3)
+    assert_images_far(f2_1, f2_3)
+    assert_images_far(f1_1, f1_3)
```

Comment:
This test will pass if `RandomizeMaterials` always does the same randomization ignoring the seed. Could you add a test to make sure 
```python
controller.step(action="SetRandomSeed", seed=41).frame
```
is different from `controller.step(action="SetRandomSeed", seed=42).frame`? I would suggest doing something like
```python
    f2_1 = controller.reset().frame
    controller.step(action="SetRandomSeed", seed=41).frame
    f2_2 = controller.step(action="RandomizeMaterials").frame
    f2_3 = controller.step(action="SetRandomSeed", seed=42).frame
    f2_4 = controller.step(action="RandomizeMaterials").frame
```
and then aserting taht `f2_2` is different is appropriately. I'd also suggest giving more informative names than `fi_j` as this test is hard to read.
as this also now tests that changing the seed

Reply:
added https://github.com/allenai/ai2thor/pull/823/commits/540e161aa9f23ed5de827c40f06b8fd63727f65b